public_suffix_list.dat

```diff
@@ -11693,4 +11695,18 @@ yolasite.com
 za.net
 za.org
 
+// Cloud DNS Ltd : http://www.cloudns.net
```

Comment:
This entire section should be moved so that it's sorted by company name, as all the other entries are